sdk-base/src/main/java/com/mapbox/maps/extension/observable/eventdata/SourceDataLoadedEventData.kt

```diff
@@ -35,5 +35,10 @@ data class SourceDataLoadedEventData(
   /**
    * The 'tile-id' property defines the tile id if the 'type' field equals 'tile'.
    */
-  @SerializedName("tile-id") val tileID: TileID?
+  @SerializedName("tile-id") val tileID: TileID?,
+
+  /**
+   * The 'data-id' property if set with the corresponding data update.
+   */
+  @SerializedName("data-id") val dataId: String?,
```

Comment:
Worth to mention that it's a new field in the data class so technically a breaking change, however should be fine since users are not suppose to create it manually or copy.